packages/plugin-mashups/test/unit/spec/mashups.js

```diff
@@ -0,0 +1,115 @@
+
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import Mashups from '../..';
+import {assert} from '@ciscospark/test-helper-chai';
+import MockSpark from '@ciscospark/test-helper-mock-spark';
+import sinon from '@ciscospark/test-helper-sinon';
+
+describe(`plugin-mashups`, () => {
+  let spark;
+  beforeEach(() => {
+    spark = new MockSpark({
+      children: {
+        mashups: Mashups
+      }
+    });
+
+    spark.request = (options) => {
+      return Promise.resolve({
+        statusCode: 204,
+        body: undefined,
+        options
+      });
+    };
+
+    sinon.spy(spark, `request`);
+  });
+
+  afterEach(() => {
+    spark.request.restore();
+  });
+
+  describe(`#create()`, () => {
+    it(`throws an error when a type is not provided`, () => assert.isRejected(spark.mashups.create({
+      displayName: `mashupName`
+    }), /options.type is required/));
+
+    it(`throws an error when a roomId is not provided`, () => assert.isRejected(spark.mashups.create({
+      displayName: `mashupName`,
+      type: `test`
+    }), /options.roomId is required/));
+
+    it(`create request sent`, () => {
+      return spark.mashups.create({type: `TYPE`, roomId: `ROOM-ID`})
```

Comment:
Take a look through your tests; most of them just return a single value, so you can write them like line 42 in this file.